modules/ting/src/Search/TingSearchCommonFields.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * @file
+ * The TingSearchCommonFields class
+ */
+
+namespace Ting\Search;
+
+/**
+ * Class TingSearchCommonFields
+ *
+ * Fields that are expected to be common across search providers.
+ *
+ * TODO: This is a good candidate for an enum.
+ *
+ * @package Ting\Search
+ */
+class TingSearchCommonFields {
+  const _PREFIX = '_field_ting_search_common_field_';
+
+  /**
+   * The date the material was added to the library.
+   */
+  const ACQUISITION_DATE = self::_PREFIX . 'acquisition_date';
```

Comment:
See previous notes about acquisition_date :-)

Reply:
What do you suggest we do here? Should all code referencing `acquisition_date` be removed from the code base? That does not seem to be within the scope of this PR.

Reply:
Lets leave it for now. It don't break anything, it just don't have any effect in DDB/FBS context.

Reply:
:+1: